ex3/Premier league.ipynb

```diff
@@ -0,0 +1,447 @@
+{
```

Comment:
<p>Nice !</p><p>For the sake of exercise, let's utilize iteration over a <code>DataFrameGroupBy</code>  object.</p><p> </p><p><code>for season, season_df in df.groupby('Season'):</code></p><p><code>  ...</code></p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/A4Vision/games-course/pull/24/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>